blog/2023-12-22-openhab-4-1-release.md

```diff
@@ -59,6 +59,33 @@ This can be used for example to add a location or equipment that does not (yet)
 To define your own semantic tags, create a YAML configuration file containing the description of all your specific semantic tags in the `$OPENHAB_CONF/tags` folder.
 You can find a [tutorial on the community forum](https://community.openhab.org/t/oh-4-1-tutorial-to-manage-custom-semantic-tags/148135).
 
+### Voice System Enhancements
+
+_Miguel Álvarez Díez, openHAB Contributor_
+
+Some minor improvements were introduced in the built-in voice system components. 
+
+The standard text interpreter now understand your Item synonyms, 
+and gives priority to an exact match on the label/synonym allowing an easier control over the command target.
```

Comment:
I assume priority to the Item name?

Reply:
I really don't know how to summarize the change.

Long explanation:

The standard interpreter matches the items by its parent names also, or allows to combine them.

Meaning in this hierarchy:

\* Group Item ("Bedroom")
** Group Item ("Light")
*** Switch Item ("Power")

The interpreter understand "Turn off the bed room", "Turn off the bed room light", "Turn off the bed room power", "Turn off the power", "Turn off the light"...  So the matching mechanism is quite "intelligent".

The problem that I found is my bed room hierarchy has another switch item:

\* Group Item ("Bed room")
** Group Item ("Light")
*** Switch Item ("Power")
** Group Item ("Lamp")
*** Switch Item ("Power")

So the phrase "Turn off the bed room" matches two different elements and can't succeed. 

But with the change now, by adding a synonym "Bed room" to the item that I want, I can decide the target of the command, as it will take priority in case multiple items are matched by the other mechanism.

Reply:
Maybe I can change to 

```
The standard text interpreter uses now the Item's synonyms metadata as alternative names, 
and allows to solving name collisions using them.
```

Reply:
I have changed it to:
```
The standard text interpreter uses now the Item's synonyms metadata as alternative names, 
and you can solve name collisions on rules using them.
```
I think it summarizes it well.